src/middleware/validators/user.validator.js

```diff
@@ -0,0 +1,22 @@
+const __ = require('../../util/response'),
+    Joi = require('joi');
+
+class UserValidator {
+
+    async user(req, res, next) {
+        const schema = Joi.object().keys({
+            email: Joi.string().email(),
+            password: Joi.string().required()
+        });
+
+        try {
+            let result = await Joi.validate(req.body, schema);
```

Comment:
Will love to know the use of await with joi.validate function

Reply:
Oh, I was just following my general coding practice. Thre is no need for Async Await. It is already blocking operation.
I'll remove it shortly.